tests/components/google_translate/test_tts.py

```diff
@@ -1,235 +1,152 @@
-
-        self.url_param["tl"] = "de"
-        aioclient_mock.get(self.url, params=self.url_param, status=200, content=b"test")
-
-        config = {tts.DOMAIN: {"platform": "google_translate", "language": "de"}}
-
-        with assert_setup_component(1, tts.DOMAIN):
-            setup_component(self.hass, tts.DOMAIN, config)
-
-        self.hass.services.call(
-            tts.DOMAIN,
-            "google_translate_say",
-            {
-                "entity_id": "media_player.something",
-                tts.ATTR_MESSAGE: "90% of I person is on front of your door.",
-            },
-        )
-        self.hass.block_till_done()
-
-        assert len(calls) == 1
-        assert len(aioclient_mock.mock_calls) == 1
-
-    @patch("gtts_token.gtts_token.Token.calculate_token", autospec=True, return_value=5)
-    def test_service_say_german_service(self, mock_calculate, aioclient_mock):
-        """Test service call say with german code in the service."""
-        calls = mock_service(self.hass, DOMAIN_MP, SERVICE_PLAY_MEDIA)
-
-        self.url_param["tl"] = "de"
-        aioclient_mock.get(self.url, params=self.url_param, status=200, content=b"test")
-
-        config = {
-            tts.DOMAIN: {"platform": "google_translate", "service_name": "google_say"}
-        }
-
-        with assert_setup_component(1, tts.DOMAIN):
-            setup_component(self.hass, tts.DOMAIN, config)
-
-        self.hass.services.call(
-            tts.DOMAIN,
-            "google_say",
-            {
-                "entity_id": "media_player.something",
-                tts.ATTR_MESSAGE: "90% of I person is on front of your door.",
-                tts.ATTR_LANGUAGE: "de",
-            },
-        )
-        self.hass.block_till_done()
-
-        assert len(calls) == 1
-        assert len(aioclient_mock.mock_calls) == 1
-
-    @patch("gtts_token.gtts_token.Token.calculate_token", autospec=True, return_value=5)
-    def test_service_say_error(self, mock_calculate, aioclient_mock):
-        """Test service call say with http response 400."""
-        calls = mock_service(self.hass, DOMAIN_MP, SERVICE_PLAY_MEDIA)
-
-        aioclient_mock.get(self.url, params=self.url_param, status=400, content=b"test")
-
-        config = {tts.DOMAIN: {"platform": "google_translate"}}
-
-        with assert_setup_component(1, tts.DOMAIN):
-            setup_component(self.hass, tts.DOMAIN, config)
-
-        self.hass.services.call(
-            tts.DOMAIN,
-            "google_translate_say",
-            {
-                "entity_id": "media_player.something",
-                tts.ATTR_MESSAGE: "90% of I person is on front of your door.",
-            },
-        )
-        self.hass.block_till_done()
-
-        assert len(calls) == 0
-        assert len(aioclient_mock.mock_calls) == 1
-
-    @patch("gtts_token.gtts_token.Token.calculate_token", autospec=True, return_value=5)
-    def test_service_say_timeout(self, mock_calculate, aioclient_mock):
```

Comment:
There is only 1 error type, that's when it raises gTTsError so I removed all other types of tests.